JustSaying/JustSayingFluently.cs

```diff
@@ -74,8 +75,16 @@ public virtual INamingStrategy GetNamingStrategy()
                     _awsClientFactoryProxy.GetAwsClientFactory().GetSnsClient(RegionEndpoint.GetBySystemName(region)),
                     Bus.SerialisationRegister);
 
-                if (!eventPublisher.Exists())
+                try
+                {
                     eventPublisher.Create();
+                }
+                catch (AuthorizationErrorException ex)
+                {
+                    Log.Error(ex, "Unable to create topic");

```

Comment:
Can we log the topic name or other identifying value?
Why squash the error and carry on?